docusaurus/docs/React/release-guides/upgrade-to-v12.mdx

```diff
@@ -61,7 +61,7 @@ Replace the render functions in your custom components with container components
 Until now, it was possible to import two stylesheets as follows:
 
 ```
-import 'stream-chat-react/dist/css/v2/index.css';
+import 'stream-chat-react/dist/css/v1/index.css';
```

Comment:
@MartinCupela I'm not sure if this is just a typo, or it was supposed to be something else completely :) I'm going to merge this, but please take a look.

Reply:
If I'm not mistaken, this import is wrong. cc: @MartinCupela

Reply:
Yes, that one is the victim of changing imports.

Reply:
What's the correct version now? :)

Reply:
I will reformulate it a bit in a separate PR